src/main/java/com/stripe/net/KeyValuePair.java

```diff
@@ -0,0 +1,11 @@
+package com.stripe.net;
+
+import java.util.AbstractMap;
+
+class KeyValuePair<K, V> extends AbstractMap.SimpleEntry<K, V> {
```

Comment:
Would you mind adding a docblock here explaining why this is needed on top of `AbstractMap.SimpleEntry`? (I don't think I completely understand myself.)

Reply:
Strictly speaking, it's not needed, the class does not implement any behavior on top of `AbstractMap.SimpleEntry`. It's purely semantics: in the context of form encoding, we're dealing with "lists of key/value pairs", not "map entries".

I'll add a comment to clarify.